### ISSUE TITLE
Whoops json error response uses status code 200

### ISSUE THREAD

**Issue by pdrosos** [zendframework/zend-expressive#475]
Hi,

I noticed that the Whoops json error response uses status code 200 when returning error.
Shouldn't it be 500 or something like that instead?

**Comment by weierophinney** [comment]
Yes, it should. I had assumed that was something done by `Zend\Stratigility\Middleware\ErrorHandler`, but, in fact, it's done within the error response generators. We have logic to do that in the production error response generator, but it's missing from the whoops variant.

Essentially, all we need to do is add an import statement:

```php
use Zend\Stratigility\Utils;
```

and one line within `__invoke()`:

```php
$response = $response->withStatus(Utils::getStatusCode($e, $response));
```

If somebody would like to tackle this, adding unit tests to verify, I'd appreciate it!

**Comment by xtreamwayz** [comment]
@weierophinney I was just looking at this.

**Comment by weierophinney** [comment]
Fixed with #476
